Modules/_asynciomodule.c

```diff
@@ -1757,17 +1753,19 @@ TaskObj_finalize(TaskObj *task)
 
     func = _PyObject_GetAttrId(task->task_loop, &PyId_call_exception_handler);
     if (func != NULL) {
-        res = PyObject_CallFunctionObjArgs(func, context, NULL);
+        PyObject *res = PyObject_CallFunctionObjArgs(func, context, NULL);
```

Comment:
(nits) While method calling API with PyID is not exported by DLL, static variable can be used.

```
static PyObject *call_exception_handler = NULL;
if (!call_exception_handler) {
    call_exception_handler = PyUnicode_InternFromString("call_exception_handler");
    if (!call_exception_handler) {
        goto finally;
    }
}

PyObject *res = PyObject_CallMethodObjArgs(task->task_loop, call_exception_handler, context, NULL);
```

This may be able to avoid creating temporary unbound method object.